Packs/Jira/Integrations/JiraV2/JiraV2.yml

```
@@ -32,10 +29,7 @@ configuration:
   defaultvalue: ""
   type: 4
   required: false
-- display: |-
-    __________________ OAuth 1.0 __________________
-
-    ConsumerKey
+- display: Consumer Key
```

```suggestion
- display: Consumer Key (OAuth 1.0)
```
what do you think? @ChanochShayner @amshamah419 @bakatzir

---

done